src/main/java/ai/deepcode/javaclient/requests/GetAnalysisRequest.java

```diff
@@ -22,17 +25,25 @@ public GetAnalysisRequest(
     List<String> limitToFiles,
     Integer severity,
     String shard,
-    boolean prioritized,
+    String ideProductName, String orgDisplayName, boolean prioritized,
```

Comment:
nitpick: maybe keep formatting consistent and have a line break between parameters.